node/package.json

```diff
@@ -23,7 +23,6 @@
     "vtex.my-account": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.my-account@1.25.0/public/@types/vtex.my-account",
     "vtex.my-account-commons": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.my-account-commons@1.6.0/public/@types/vtex.my-account-commons",
     "vtex.render-runtime": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.render-runtime@8.132.4/public/@types/vtex.render-runtime",
-    "vtex.return-app": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.return-app@3.0.0-beta.12/public/@types/vtex.return-app",
```

Comment:
Since this changes the types, I think it's worth releasing a new beta, install on your workspace, unlink the project and run `vtex setup --typings` to get the correct package for this types. I know it's kind of annoying, but needed if we want to deploy from github.